### DISCUSSION TITLE
Got asyncio.exceptions.CancelledError when ctrl+c the application

### DISCUSSION THREAD

**Discussion by igormcsouza** [fastapi/fastapi#6615] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
@app.post("/catbot")
async def catbot(body: CatBotBody):
    answer = await chatbot.get_response(body.question)

    return {"msg": answer}
```

### Description

When I cancel hitting ctrl+c I got the error msg.

```
^CINFO:     Shutting down
INFO:     Finished server process [46235]
ERROR:    Traceback (most recent call last):
  File "/home/lead/souza/kitty-api/.venv/lib/python3.8/site-packages/starlette/routing.py", line 624, in lifespan
    await receive()
  File "/home/lead/souza/kitty-api/.venv/lib/python3.8/site-packages/uvicorn/lifespan/on.py", line 135, in receive
    return await self.receive_queue.get()
  File "/usr/lib/python3.8/asyncio/queues.py", line 163, in get
    await getter
asyncio.exceptions.CancelledError
```

Here is a link to the repo where this problem happens. https://github.com/igormcsouza/kitty-api/blob/master/api/__init__.py

### Operating System

Linux

### FastAPI Version

0.70.0

### Python Version

3.8

### Additional Context

The code I'm runing is a very simple post request. Just wrote it!

**Comment by Kludex** [comment, marked as answer]
This is a Uvicorn issue, not a FastAPI one.

**Comment by igormcsouza** [comment]
> This is a Uvicorn issue, not a FastAPI one.

You might be right! I'm going to close here as soon as I create/find a issue there!

**Comment by igormcsouza** [comment]
https://github.com/encode/uvicorn/issues/1160

that's the issue! thanks!!

**Comment by afparsons** [comment]
For passers-by: I encountered a similar issue when setting spaCy's `Language.pipe` `n_process` parameter to `> 1`.
